static/app/views/settings/dynamicSampling/projectsPreviewTable.tsx

```diff
@@ -0,0 +1,234 @@
+import {useDebouncedValue} from 'sentry/utils/useDebouncedValue';
+import useProjects from 'sentry/utils/useProjects';
+import {dynamicSamplingForm} from 'sentry/views/settings/dynamicSampling/dynamicSamplingForm';
+import {balanceSampleRate} from 'sentry/views/settings/dynamicSampling/utils/rebalancing';
+
+const {useFormField} = dynamicSamplingForm;
+
+// TODO(aknaus): Switch to c:spans/count_per_root_project@none once available
+const SPANS_COUNT_METRIC: MRI = `c:transactions/count_per_root_project@none`;
+const metricsQuery: MetricsQueryApiQueryParams[] = [
+  {
+    mri: SPANS_COUNT_METRIC,
+    aggregation: 'count',
+    name: 'spans',
+    groupBy: ['project'],
+    orderBy: 'desc',
+  },
+];
+
+export function ProjectsPreviewTable() {
+  const {projects, fetching} = useProjects();
+  const [tableSort, setTableSort] = useState<'asc' | 'desc'>('desc');
+  const [period, setPeriod] = useState<'24h' | '30d'>('24h');
+  const {value: targetSampleRate} = useFormField('targetSampleRate');
+
+  const {data, isPending, isError, refetch} = useMetricsQuery(
+    metricsQuery,
+    {
+      datetime: {
+        start: null,
+        end: null,
+        utc: true,
+        period,
+      },
+      environments: [],
+      projects: [],
+    },
+    {
+      includeSeries: false,
+      interval: period === '24h' ? '1h' : '1d',
+    }
+  );
+
+  const projectBySlug = useMemo(
+    () =>
+      projects.reduce((acc, project) => {
+        acc[project.slug] = project;
+        return acc;
+      }, {}),
+    [projects]
+  );
+
+  const items = useMemo(
+    () =>
+      (data?.data[0] ?? [])
+        .map(item => ({
+          id: item.by.project,
+          project: projectBySlug[item.by.project],
+          count: item.totals,
+          // This is a placeholder value to satisfy typing
+          // the actual value is calculated in the balanceSampleRate function
+          sampleRate: 1,
+        }))
+        // Remove items where we cannot match the project
+        .filter(item => item.project),
+    [data?.data, projectBySlug]
+  );
+
+  const debouncedTargetSampleRate = useDebouncedValue(
+    targetSampleRate,
+    // For longer lists we debounce the input to avoid too many re-renders
+    items.length > 100 ? 200 : 0
+  );
+
+  const {balancedItems} = useMemo(() => {
+    const targetRate = Math.min(100, Math.max(0, Number(debouncedTargetSampleRate) || 0));
+    return balanceSampleRate({
+      targetSampleRate: targetRate / 100,
+      items,
+    });
+  }, [debouncedTargetSampleRate, items]);
+
+  if (isError) {
+    return <LoadingError onRetry={() => refetch()} />;
+  }
+
+  return (
+    <ProjectsTable
+      stickyHeaders
+      emptyMessage={t('No active projects found in the selected period.')}
+      isEmpty={!items.length}
+      isLoading={isPending || fetching}
+      headers={[
+        t('Project'),
+        <div key={'spans'} style={{display: 'flex', alignItems: 'center'}}>
```

Comment:
should we use styled css? 
```suggestion
        <div key='spans' style={{display: 'flex', alignItems: 'center'}}>
```